packages/tasks/src/model-data.ts

```diff
@@ -40,21 +40,29 @@ export interface ModelData {
 	/**
 	 * this dictionary has useful information about the model configuration
 	 */
-	config?: Record<string, unknown> & {
-		adapter_transformers?: { model_class?: string; model_name?: string };
+	config?: {
 		architectures?: string[];
+		tokenizer_config?: TokenizerConfig;
+		adapter_transformers?: {
+			model_name?: string;
+			model_class?: string;
+		};
+		diffusers?: {
+			_class_name?: string;
+		};
 		sklearn?: {
-			filename?: string;
+			model?: {
+				file?: string;
+			};
 			model_format?: string;
 		};
 		speechbrain?: {
-			interface?: string;
+			speechbrain_interface?: string;
```

Comment:
Btw I just realized the `speechbrain` API needs some additional fields https://github.com/huggingface/api-inference-community/blob/main/docker_images/speechbrain/app/common.py#L39

Reply:
`vocoder_model_id` is also in there, i'll add it to types

Reply:
see 14ffcb7b740378e16a3a2bdf93325235528f70d7

Reply:
Ah nice, nothing to worry then!